src/index.tsx

```diff
@@ -51,6 +62,11 @@ export default function Command() {
             maxCapacity={state.batteryRegistry["AppleRawMaxCapacity"] || state.batteryRegistry["MaxCapacity"]}
           />
           <CycleCountItem cycles={state.batteryRegistry["CycleCount"]} />
+          <BatteryHealthItem
+            health={
+              state.batteryInfo["_items"][0]["sppower_battery_health_info"]["sppower_battery_health_maximum_capacity"]
```

Comment:
The value `sppower_battery_health_maximum_capacity` from the system_profiler is only available on a recent macOS. This is fine, but we should take care that no error message is thrown on systems which does not have that value. What do you think? :)